framework/PostProcessors/FTgenerator.py

```diff
@@ -0,0 +1,288 @@
+# Copyright 2017 Battelle Energy Alliance, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Created on March 25, 2020
+
+@author: mandd
+"""
+
+#External Modules---------------------------------------------------------------
+import numpy as np
+import xml.etree.ElementTree as ET
+import math
+#External Modules End-----------------------------------------------------------
+
+#Internal Modules---------------------------------------------------------------
+from .PostProcessor import PostProcessor
+from utils import utils, xmlUtils
+from utils import InputData, InputTypes
+from sympy.logic import SOPform, POSform
+from itertools import product
+import Runners
+#Internal Modules End-----------------------------------------------------------
+
+class FTgenerator(PostProcessor):
+  """
+    This postprocessor is designed to create a fault tree in OpenPSA format from a set of simulation data
+    The postprocessor acts only on PointSet and return a reduced PointSet and print of file the FT in OpenPSA format
+  """
+  def __init__(self, messageHandler):
+    """
+      Constructor
+      @ In, messageHandler, MessageHandler, message handler object
+      @ Out, None
+    """
+    PostProcessor.__init__(self, messageHandler)
+    self.inputVars = None    # variable associated with the lower limit of the value dimension
```

Comment:
variable associated with the lower limit of the value dimension -->list of variables associated with the lower limits of the value dimension